tests/routes/pro/get_venues_test.py

```diff
@@ -1,108 +1,276 @@
+
+        other_offerer_not_validated = offers_factories.OffererFactory(validationToken="other_token")
+        offers_factories.UserOffererFactory(user=other_user_offerer.user, offerer=other_offerer_not_validated)
+        other_venue_not_validated = offers_factories.VenueFactory(
+            name="other_venue_not_validated",
+            managingOfferer=other_offerer_not_validated,
+        )
+
+        return {
+            "venue": venue,
+            "venue_not_validated": venue_not_validated,
+            "other_venue": other_venue,
+            "other_venue_not_validated": other_venue_not_validated,
+        }
+
+    @pytest.mark.usefixtures("db_session")
+    def test_get_all_venues(self, app):
+        # given
+        admin = users_factories.UserFactory(isBeneficiary=False, isAdmin=True)
+        venues = self._setup_venues_for_users()
+
+        # when
+        response = TestClient(app.test_client()).with_auth(admin.email).get("/venues")
+
+        # then
+        assert response.status_code == 200
+        assert len(response.json) == 4
+
+        venue_ids = [venue["id"] for venue in response.json]
+        assert humanize(venues["venue"].id) in venue_ids
+        assert humanize(venues["venue_not_validated"].id) in venue_ids
+        assert humanize(venues["other_venue"].id) in venue_ids
+        assert humanize(venues["other_venue_not_validated"].id) in venue_ids
+
+    @pytest.mark.usefixtures("db_session")
+    def test_get_all_validated_venues(self, app):
+        # given
+        admin = users_factories.UserFactory(isBeneficiary=False, isAdmin=True)
+        venues = self._setup_venues_for_users()
+
+        # when
+        response = TestClient(app.test_client()).with_auth(admin.email).get("/venues?validated=true")
+
+        # then
+        assert response.status_code == 200
+        assert len(response.json) == 2
+
+        venue_ids = [venue["id"] for venue in response.json]
+        assert humanize(venues["venue"].id) in venue_ids
+        assert humanize(venues["other_venue"].id) in venue_ids
+
+    @pytest.mark.usefixtures("db_session")
+    def test_get_all_not_validated_venues(self, app):
+        # given
+        admin = users_factories.UserFactory(isBeneficiary=False, isAdmin=True)
+        venues = self._setup_venues_for_users()
+
+        # when
+        response = TestClient(app.test_client()).with_auth(admin.email).get("/venues?validated=false")
+
+        # then
+        assert response.status_code == 200
+        assert len(response.json) == 2
+
+        venue_ids = [venue["id"] for venue in response.json]
+        assert humanize(venues["venue_not_validated"].id) in venue_ids
+        assert humanize(venues["other_venue_not_validated"].id) in venue_ids
+
+
+class Returns403:
+    @pytest.mark.usefixtures("db_session")
+    def when_current_user_doesnt_have_rights(self, app):
+        # given
+        offerer = offers_factories.OffererFactory()
+        user = users_factories.UserFactory(isBeneficiary=False)
```

Comment:
```suggestion
        pro_user = users_factories.UserFactory(isBeneficiary=False)
```
Pour correspondre à ce que tu as mis plus haut